cpp/src/aztec/dsl/turbo_proofs/CMakeLists.txt

```diff
@@ -0,0 +1,5 @@
+barretenberg_module(
+    turbo_proofs
+    acir_format
+    plonk
+)
```

Comment:
I have no idea what all the link dependencies should be here.

Reply:
@kevaundray Who would be able to determine which modules should be linked here?

Reply:
I think David will be helpful here

Reply:
The process I've used in the past is:
1. Any modules called out in headers
2. Don't need to include lower-level modules that are already included by another module dependency here (e.g. maybe plonk includes stdlib_<something>)
3. Omit header-only modules